core-services/prow/02_config/_config.yaml

```diff
@@ -30,6 +24,29 @@ deck:
       - bparees
       - AlexNPavel
       - jupierce
+  - repo: openshift/openshift-tests-private
+    rerun_auth_configs:
+      github_team_slugs:
+      - org: openshift
+        slug: team-qe
+  - repo: openshift/verification-tests
+    rerun_auth_configs:
+      github_team_slugs:
+      - org: openshift
+        slug: team-qe
+  - repo: smg247/ci-docs
+    rerun_auth_configs:
+      github_team_slugs:
+      - org: smg247-org
+        slug: sample
+  external_agent_logs:
+  - agent: jenkins
+    selector: master=ci.openshift.redhat.com
+    url_template: http://jenkins-operator/job/{{.Spec.Job}}/{{.Status.JenkinsBuildID}}/consoleText
+  hidden_repos:
+  - openshift/ose
+  - openshift/online-registration
+  rerun_auth_configs:
```

Comment:
do we still need this one?

Reply:
That means `marshall` function is wrong at upstream.

Reply:
https://github.com/kubernetes/test-infra/blob/master/config/prow/config.yaml#L166 Upstream works.
[11:30](https://coreos.slack.com/archives/D02AJ9F8TFY/p1657294240558739)
The bug is in our tool determinize-prow-config